src/internal/packager2/remove.go

```diff
@@ -167,5 +167,6 @@ func Remove(ctx context.Context, opt RemoveOptions) error {
 		}
 	}
 
+	l.Info("package successfully removed", "name", pkg.Metadata.Name)
```

Comment:
Adding this in here as an aside. Someone gave me feedback that `zarf package remove` feels strange since it doesn't give any confirmation that the remove happened successfully, I would agree